frameworks/ruby/rails6/run.sh

```diff
@@ -17,4 +17,4 @@ rake 'user:get[101]'
 rake user:create_fail || true
 rake user:list | tail
 
-python testingGuides.py
+source ActiveRecordMigrations.sh
```

Comment:
This should be executed instead of sourced.

Reply:
Fixed